components/sections/Layout/Footer.tsx

```diff
@@ -2,9 +2,20 @@ import { useTranslation } from 'next-i18next';
 import Link from 'next/link';
 import Image from 'next/image';
 import { FOOTER_NAVIGATION } from 'data/footerNavigation';
+import { getCountryVariant } from 'utils/locales';
+import { useLanguage } from 'utils/useLanguageHook';
 
 export const Footer = () => {
   const { t } = useTranslation();
+  const { currentLanguage } = useLanguage();
+
+  const footerNavigationLinks = Object.entries(FOOTER_NAVIGATION)
```

Comment:
why not just `footerNavigationLinks = FOOTER_NAVIGATION[getCountryVariant()][currentLanguage]`